techniques/html/H48.html

```diff
@@ -128,17 +132,17 @@ <h3>Example 2: A grocery list </h3>
 &lt;/ul&gt;</code></pre>
       </section>
       <section class="example" id="example-3-a-word-and-its-definition">
-         <h3>Example 3: A word and its definition</h3>
-         <p>This example uses a description list to group a definition with the term that is being defined.</p>
+         <h3>事例 3: 材料のリスト</h3>
```

Comment:
原文が「A word and its definition」なので、集成しました。
```suggestion
         <h3>事例 3: 単語及びその定義</h3>
```